tests/test_core.py

```diff
@@ -100,7 +100,7 @@ def test_syllabify_exceptions_rh_dipthong():
 
 def test_syllabify_tl():
     word = "atlante"
-    output = ['at', 'lan', 'te']
+    output = ['a', 'tlan', 'te']
```

Comment:
Why?

Reply:
I have decided to join the 'tl' cluster because everyone (DIRAE, PERKINS, EDUCOLINGO) do it that way,

Reply:
Please, add a note explaining that.

Reply:
Where, exactly? With the regexes for syllabification?

Reply:
That's a good place, yes.